pkg/api/volume_resp.go

```diff
@@ -73,6 +73,15 @@ type VolumeStatusResp struct {
 	Size   SizeInfo   `json:"size"`
 }
 
+// VolumeOptionGetResp is the response sent for a volume option get request
+type VolumeOptionGetResp struct {
+	OptName      string `json:"name"`
+	Value        string `json:"value"`
+	Modified     bool   `json:"modified"`
+	DefaultValue string `json:"default_value"`
```

Comment:
Use hyphen, not underscores for json strings.

Reply:
done